<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBratMainWindow</class>
 <widget class="QMainWindow" name="CBratMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brat MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="mMainMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>16</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>Work&amp;space</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Close_Workspace"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="action_Import_Workspace"/>
    <addaction name="action_Rename_Workspace"/>
    <addaction name="action_Delete_Workspace"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Open_View"/>
    <addaction name="action_Views_List"/>
    <addaction name="action_Close_All"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="separator"/>
    <addaction name="action_Test"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Zoom_In"/>
    <addaction name="action_Zoom_Out"/>
    <addaction name="action_Re_center"/>
    <addaction name="action_Refresh_Map"/>
    <addaction name="separator"/>
    <addaction name="action_Satellite_Tracks"/>
    <addaction name="action_Save_Map_Image"/>
    <addaction name="separator"/>
    <addaction name="action_Full_Screen"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_One_Click"/>
    <addaction name="action_Workspace_Tree"/>
    <addaction name="action_Launch_Scheduler"/>
    <addaction name="separator"/>
    <addaction name="action_Dataset"/>
    <addaction name="action_Filter"/>
    <addaction name="action_Operations"/>
    <addaction name="separator"/>
    <addaction name="action_Options"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mMainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_One_Click"/>
   <addaction name="action_Launch_Scheduler"/>
   <addaction name="separator"/>
   <addaction name="action_Workspace_Tree"/>
  </widget>
  <widget class="QStatusBar" name="mMainStatusBar"/>
  <widget class="QToolBar" name="mMainToolsToolBar">
   <property name="windowTitle">
    <string>Map Toolbar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Zoom_In"/>
   <addaction name="action_Zoom_Out"/>
   <addaction name="action_Re_center"/>
   <addaction name="action_Refresh_Map"/>
   <addaction name="separator"/>
   <addaction name="action_Satellite_Tracks"/>
   <addaction name="action_Save_Map_Image"/>
   <addaction name="separator"/>
   <addaction name="action_Full_Screen"/>
   <addaction name="separator"/>
   <addaction name="action_Options"/>
   <addaction name="separator"/>
   <addaction name="action_Test"/>
  </widget>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/0.png</normaloff>:/images/alpha-numeric/0.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit brat</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/1.png</normaloff>:/images/alpha-numeric/1.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open existing workspace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/2.png</normaloff>:/images/alpha-numeric/2.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="toolTip">
    <string>Create a new workspace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/3.png</normaloff>:/images/alpha-numeric/3.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the workspace to disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/BratIcon.png</normaloff>:/images/BratIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_Close_All">
   <property name="text">
    <string>Close &amp;All</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/6.png</normaloff>:/images/alpha-numeric/6.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options...</string>
   </property>
   <property name="toolTip">
    <string>Edit application settings</string>
   </property>
  </action>
  <action name="action_Test">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/test.png</normaloff>:/images/test.png</iconset>
   </property>
   <property name="text">
    <string>Test...</string>
   </property>
  </action>
  <action name="action_One_Click">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/7.png</normaloff>:/images/alpha-numeric/7.png</iconset>
   </property>
   <property name="text">
    <string>One-&amp;Click...</string>
   </property>
   <property name="toolTip">
    <string>Group actions as a single operation under a single name</string>
   </property>
  </action>
  <action name="action_Launch_Scheduler">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/8.png</normaloff>:/images/alpha-numeric/8.png</iconset>
   </property>
   <property name="text">
    <string>Launch Scheduler</string>
   </property>
   <property name="toolTip">
    <string>Launch Scheduler application</string>
   </property>
  </action>
  <action name="action_Workspace_Tree">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/9.png</normaloff>:/images/alpha-numeric/9.png</iconset>
   </property>
   <property name="text">
    <string>Workspace Tree...</string>
   </property>
   <property name="toolTip">
    <string>See the objects tree of the loaded workspace</string>
   </property>
  </action>
  <action name="action_Save_Map_Image">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/a.png</normaloff>:/images/alpha-numeric/a.png</iconset>
   </property>
   <property name="text">
    <string>Save Map Image...</string>
   </property>
   <property name="toolTip">
    <string>Save map to file</string>
   </property>
  </action>
  <action name="action_Refresh_Map">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/b.png</normaloff>:/images/alpha-numeric/b.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh main map</string>
   </property>
  </action>
  <action name="action_Graphic_Settings">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/c.png</normaloff>:/images/alpha-numeric/c.png</iconset>
   </property>
   <property name="text">
    <string>Graphic Settings...</string>
   </property>
   <property name="toolTip">
    <string>Adjust graphic settings</string>
   </property>
  </action>
  <action name="action_Full_Screen">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/d.png</normaloff>:/images/alpha-numeric/d.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggle full screen mode.&lt;/p&gt;&lt;p&gt;Use the &lt;span style=&quot; font-weight:600;&quot;&gt;F11&lt;/span&gt; key to switch modes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="action_Re_center">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/e.png</normaloff>:/images/alpha-numeric/e.png</iconset>
   </property>
   <property name="text">
    <string>Re-center</string>
   </property>
   <property name="toolTip">
    <string>Bring view to original position and size</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="action_Zoom_In">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/f.png</normaloff>:/images/alpha-numeric/f.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom map in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_Zoom_Out">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/5.png</normaloff>:/images/alpha-numeric/5.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom map out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_Operations">
   <property name="text">
    <string>&amp;Operations</string>
   </property>
  </action>
  <action name="action_Dataset">
   <property name="text">
    <string>&amp;Dataset</string>
   </property>
  </action>
  <action name="action_Filter">
   <property name="text">
    <string>&amp;Filter</string>
   </property>
  </action>
  <action name="action_Import_Workspace">
   <property name="text">
    <string>&amp;Import...</string>
   </property>
   <property name="toolTip">
    <string>Import existing workspace objects</string>
   </property>
  </action>
  <action name="action_Rename_Workspace">
   <property name="text">
    <string>&amp;Rename...</string>
   </property>
   <property name="toolTip">
    <string>Rename the loaded workspace
The workspace directory will not change</string>
   </property>
  </action>
  <action name="action_Delete_Workspace">
   <property name="text">
    <string>&amp;Delete...</string>
   </property>
   <property name="toolTip">
    <string>Delete the loaded workspace</string>
   </property>
  </action>
  <action name="action_Close_Workspace">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/__q.png</normaloff>:/images/alpha-numeric/__q.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close and unload the workspace</string>
   </property>
  </action>
  <action name="action_Satellite_Tracks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/alpha-numeric/r.png</normaloff>:/images/alpha-numeric/r.png</iconset>
   </property>
   <property name="text">
    <string>Satellite Tracks</string>
   </property>
   <property name="toolTip">
    <string>Enable automatic satellite track drawing when selected dataset changes</string>
   </property>
  </action>
  <action name="action_Views_List">
   <property name="icon">
    <iconset resource="Resources/images/qgis-themes-2.qrc">
     <normaloff>:/images/themes/default/propertyicons/diagram.png</normaloff>:/images/themes/default/propertyicons/diagram.png</iconset>
   </property>
   <property name="text">
    <string>&amp;List...</string>
   </property>
   <property name="toolTip">
    <string>List all open windows and select or close views</string>
   </property>
  </action>
  <action name="action_Open_View">
   <property name="icon">
    <iconset resource="../Common/Resources/brat.qrc">
     <normaloff>:/images/OSGeo/open.png</normaloff>:/images/OSGeo/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Select a view from the list of all workspace views</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/images/qgis-themes-2.qrc"/>
  <include location="../Common/Resources/brat.qrc"/>
  <include location="../Common/Resources/brat.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>CBratMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
